Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
 
 members = [
-    "crd", "operator", "server"
+    "rust/crd", "rust/operator", "rust/operator-lib"
```

Comment:
This could just be `"rust/*"`.

Reply:
That is a good point and we'd prefer that, but at the moment our release helper tool doesn't work with that syntax. Once that has been fixed we'll change this.

https://github.com/stackabletech/stackable-utils/issues/6